bindings/python/native/src/client/high_level_api.rs

```diff
@@ -0,0 +1,285 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::client::{error::Result, AddressBalancePair, Client, Input, Message, MessageMetadata, Output};
+use iota::{
+    Bech32Address as RustBech32Address, MessageId as RustMessageId, Seed as RustSeed,
+    TransactionId as RustTransationId, UTXOInput as RustUTXOInput,
+};
+use pyo3::prelude::*;
+use std::{
+    convert::{From, Into, TryInto},
+    str::FromStr,
+};
+
+/// General high level APIs
+#[pymethods]
+impl Client {
+    fn send(
+        &self,
+        seed: Option<String>,
+        account_index: Option<usize>,
+        initial_address_index: Option<usize>,
+        inputs: Option<Vec<Input>>,
+        outputs: Option<Vec<Output>>,
+        index: Option<&str>,
+        data: Option<Vec<u8>>,
+        data_str: Option<String>,
+        parent: Option<&str>,
+        network_id: Option<u64>,
+    ) -> Result<Message> {
+        let mut send_builder = self.client.send();
+        if let Some(account_index) = account_index {
+            send_builder = send_builder.with_account_index(account_index);
+        }
+        if let Some(initial_address_index) = initial_address_index {
+            send_builder = send_builder.with_initial_address_index(initial_address_index);
+        }
+        if let Some(inputs) = inputs {
```

Comment:
Custom range for the inputs is missing

Reply:
Thank you for identifying this. Added error handling [here](https://github.com/iotaledger/iota.rs/blob/c1aeb5324724b9ce83ef4f6072c7403d9803a597/bindings/python/native/src/client/high_level_api.rs#L36-L42) and supported user inputs [here](https://github.com/iotaledger/iota.rs/blob/c1aeb5324724b9ce83ef4f6072c7403d9803a597/bindings/python/native/src/client/high_level_api.rs#L58-L60)